nemo/collections/common/parts/multi_layer_perceptron.py

```diff
@@ -0,0 +1,58 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import pytorch_lightning as pl
+import torch
+
+
+class MultiLayerPerceptron(pl.LightningModule):
```

Comment:
Seems like this should inherit from "nemo.core.NeuralModule"

Reply:
it's not a separate module, it's a 'part' that is used in one of the modules - TokenClassifier. I don't see how we can add restore_from and save to it?